contrib/macdeploy/macdeployqtplus

```diff
@@ -719,14 +718,14 @@ if config.dmg is not None:
         def f(*popenargs, **kwargs):
             if 'stdout' in kwargs:
                 raise ValueError('stdout argument not allowed, it will be overridden.')
-            process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, **kwargs)
+            process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, universal_newlines=True, **kwargs)
             output, unused_err = process.communicate()
             retcode = process.poll()
             if retcode:
                 cmd = kwargs.get("args")
                 if cmd is None:
                     cmd = popenargs[0]
-                raise CalledProcessError(retcode, cmd)
```

Comment:
This was just broken before, I guess? I don't see how it could have worked.

Reply:
indeed